eggo/fabric_cli.py

```diff
@@ -124,35 +125,49 @@ def do():
 
     execute(do, hosts=get_worker_hosts())
 
+def sudo(command, **kwargs):
```

Comment:
I tried that first, but it turns out that the environment (e.g. `PATH`) is different under `sudo` so it doesn't work.